src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -127,6 +132,11 @@ public abstract class WebRequestPSCmdlet : PSCmdlet
         /// </summary>
         private bool _resumeSuccess = false;
 
+        /// <summary>
+        /// The Dispose() method has already been called to cleanup Disposable fields.
+        /// </summary>
+        private bool _disposedValue = false;
```

Comment:
Please make the comment better.

Reply:
I have changed the comment, but not sure if it qualifies as better.